packages/mui-varnish/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@allenai/mui-varnish",
+  "version": "5.11.0",
```

Comment:
Is this right? I'd expect it to be `0.1.1` or something that indicates it's in alpha.